src/Pages/CategoriesPage.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+const CategoriesPage = () => {
+    return (
+        <div>
+            <button className='border-2 border-black' type='button'>Check status</button>
+        </div>
+    )
+}
```

Comment:
- [ ]  @gauravgangwarweb great work on your component however am getting the error below, its caused by having a return statement yet you don't have any methods inside the component, kindly check my suggestion in the BooksPage component

![image](https://user-images.githubusercontent.com/61408860/213488799-64dd66e1-0e06-4a89-ad67-12eacc5dc40a.png)